addons/hr_evaluation/hr_department.py

```diff
@@ -0,0 +1,28 @@
+# -*- encoding: utf-8 -*-
+
+from openerp import models, fields, api
+
+
+class hr_department(models.Model):
+    _inherit = 'hr.department'
+
+    @api.multi
+    def _interview_request_count(self):
+        Interview = self.env['hr.evaluation.interview']
+        for department in self:
+            department.interview_request_count = Interview.search_count([
```

Comment:
@jem-odoo  Here read_group does not seem feasible as we do not have straight relationship with department object.